showcase/app/templates/components/app-header/index.hbs

```diff
@@ -241,6 +241,91 @@
     </SG.Item>
   </Shw::Grid>
 
+  <Shw::Divider @level="2" />
+
+  <Shw::Text::H3>With text</Shw::Text::H3>
+
+  <Shw::Grid @columns={{1}} {{style gap="2rem"}} as |SG|>
```

Comment:
[general consideration] no need to change it here, but in general it would  be better to avoid using a grid to achieve a simple vertical stacked layout, and and use `<Shw::Grid @direction="column">`instead (CSS grid comes with (rare) potential side effects, that we want to avoid)

Reply:
Good suggestion, I've just replaced all of the grids with flex instead since these are basically just single column layouts. I think this was just a holdover from the previous template.